src/oop_ext/interface/_interface.py

```diff
@@ -191,7 +191,7 @@ def _IsClass(obj):
     return isinstance(obj, type)
 
 
-def IsImplementation(class_or_instance, interface, *, requires_declaration=False):
+def IsImplementation(class_or_instance, interface, *, requires_declaration=True):
```

Comment:
Would be nice to have it clear on docs what would happen if `requires_declaration=False`, since historically we always expect the Interface to be declared (so an ancient `Interfaces` user would know why we added this flag).

suggestion:
`...to be considered an implementation of `interface`. Otherwise it'd only check if `class_or_interface` implements all methods defined on `interface`.

Reply:
Done